### PR TITLE
Update outdated files in the dev-1.20-ko.1 branch (1)

### LINKED ISSUE #25581: Outdated files in the dev-1.20-ko.1 branch. (1)
Outdated files in the dev-1.20-ko.1 branch (1)
This is a part of #25521 .

55 files to be modified
- [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 20 insertions(+)
- [x] content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/logging.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [ ] content/en/docs/concepts/cluster-administration/system-metrics.md 1 file changed, 23 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 4 insertions(+)
- [x] content/en/docs/concepts/configuration/secret.md 1 file changed, 12 insertions(+), 8 deletions(-)
- [x] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md 1 file changed, 2 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 39 insertions(+), 19 deletions(-)
- [x] content/en/docs/concepts/overview/working-with-objects/labels.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/policy/pod-security-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [ ] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 43 insertions(+)
- [x] content/en/docs/concepts/scheduling-eviction/pod-overhead.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 1 insertion(+), 5 deletions(-)
- [ ] content/en/docs/concepts/services-networking/dual-stack.md 1 file changed, 151 insertions(+), 21 deletions(-)
- [ ] content/en/docs/concepts/services-networking/endpoint-slices.md 1 file changed, 51 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 28 insertions(+), 28 deletions(-)
- [ ] content/en/docs/concepts/services-networking/service.md 1 file changed, 66 insertions(+), 48 deletions(-)
- [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 34 insertions(+), 27 deletions(-)
- [x] content/en/docs/concepts/storage/volume-snapshot-classes.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/storage/volume-snapshots.md 1 file changed, 7 insertions(+), 7 deletions(-)
- [x] content/en/docs/concepts/storage/volumes.md 1 file changed, 6 insertions(+)
- [x] content/en/docs/concepts/workloads/controllers/cron-jobs.md 1 file changed, 8 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 6 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 16 insertions(+), 5 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/ephemeral-containers.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 19 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/access-authn-authz/service-accounts-admin.md 1 file changed, 14 insertions(+), 1 deletion(-)
- [ ] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 81 insertions(+), 26 deletions(-)
- [ ] content/en/docs/reference/command-line-tools-reference/kube-proxy.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/reference/kubectl/conventions.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/kubectl/kubectl.md 1 file changed, 9 insertions(+), 3 deletions(-)
- [x] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [ ] content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md 1 file changed, 34 insertions(+), 65 deletions(-)
- [x] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [ ] content/en/docs/setup/release/notes.md 1 file changed, 1656 insertions(+), 2119 deletions(-)
- [ ] content/en/docs/tasks/administer-cluster/highly-available-master.md 1 file changed, 33 insertions(+)
- [ ] content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md 1 file changed, 74 insertions(+), 19 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md 1 file changed, 8 insertions(+), 8 deletions(-)
- [ ] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md 1 file changed, 92 insertions(+), 213 deletions(-)
- [ ] content/en/docs/tasks/network/validate-dual-stack.md 1 file changed, 96 insertions(+), 23 deletions(-)
- [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 69 insertions(+)
- [x] content/en/examples/application/job/cronjob.yaml 1 file changed, 1 insertion(+)
- [x] content/en/examples/application/mysql/mysql-configmap.yaml 1 file changed, 4 insertions(+), 4 deletions(-)
- [x] content/en/examples/application/mysql/mysql-services.yaml 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/application/mysql/mysql-statefulset.yaml 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/examples/service/networking/dual-stack-default-svc.yaml 1 file changed, 2 insertions(+), 1 deletion(-)

1 files to be deleted
- [x] content/en/docs/concepts/workloads/pods/podpreset.md

/language ko
/assign

### PR DESCRIPTION
**Pull request by pjhwa**
#25521 
Closes #25581 

Update outdated files in the dev-1.20-ko.1 branch.

36/52 files are modified
- [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 20 insertions(+)
- [x] content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [ ] content/en/docs/concepts/cluster-administration/system-metrics.md 1 file changed, 23 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 4 insertions(+)
- [x] content/en/docs/concepts/configuration/secret.md 1 file changed, 12 insertions(+), 8 deletions(-)
- [x] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 2 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md 1 file changed, 2 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 39 insertions(+), 19 deletions(-)
- [x] content/en/docs/concepts/overview/working-with-objects/labels.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/policy/pod-security-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [ ] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 43 insertions(+)
- [x] content/en/docs/concepts/scheduling-eviction/pod-overhead.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 1 insertion(+), 5 deletions(-)
- [ ] content/en/docs/concepts/services-networking/dual-stack.md 1 file changed, 151 insertions(+), 21 deletions(-)
- [ ] content/en/docs/concepts/services-networking/endpoint-slices.md 1 file changed, 51 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 28 insertions(+), 28 deletions(-)
- [ ] content/en/docs/concepts/services-networking/service.md 1 file changed, 66 insertions(+), 48 deletions(-)
- [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 34 insertions(+), 27 deletions(-)
- [x] content/en/docs/concepts/storage/volume-snapshot-classes.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/storage/volume-snapshots.md 1 file changed, 7 insertions(+), 7 deletions(-)
- [x] content/en/docs/concepts/storage/volumes.md 1 file changed, 6 insertions(+)
- [x] content/en/docs/concepts/workloads/controllers/cron-jobs.md 1 file changed, 8 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 6 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 16 insertions(+), 5 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/ephemeral-containers.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 19 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/access-authn-authz/service-accounts-admin.md 1 file changed, 14 insertions(+), 1 deletion(-)
- [ ] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 81 insertions(+), 26 deletions(-)
- [ ] content/en/docs/reference/command-line-tools-reference/kube-proxy.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/reference/kubectl/conventions.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/kubectl/kubectl.md 1 file changed, 9 insertions(+), 3 deletions(-)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [ ] content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md 1 file changed, 34 insertions(+), 65 deletions(-)
- [x] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [ ] content/en/docs/setup/release/notes.md 1 file changed, 1656 insertions(+), 2119 deletions(-)
- [ ] content/en/docs/tasks/administer-cluster/highly-available-master.md 1 file changed, 33 insertions(+)
- [ ] content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md 1 file changed, 74 insertions(+), 19 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md 1 file changed, 8 insertions(+), 8 deletions(-)
- [ ] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md 1 file changed, 92 insertions(+), 213 deletions(-)
- [ ] content/en/docs/tasks/network/validate-dual-stack.md 1 file changed, 96 insertions(+), 23 deletions(-)
- [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 69 insertions(+)
- [x] content/en/examples/application/job/cronjob.yaml 1 file changed, 1 insertion(+)
- [x] content/en/examples/application/mysql/mysql-configmap.yaml 1 file changed, 4 insertions(+), 4 deletions(-)
- [x] content/en/examples/application/mysql/mysql-services.yaml 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/application/mysql/mysql-statefulset.yaml 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/examples/service/networking/dual-stack-default-svc.yaml 1 file changed, 2 insertions(+), 1 deletion(-)

1 file is deleted
- [x] content/en/docs/concepts/workloads/pods/podpreset.md

3 files are already updated in Korean documents
- [x] content/en/docs/concepts/cluster-administration/logging.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 3 insertions(+), 3 deletions(-)

### CODE REVIEW

File: content/ko/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -105,29 +96,44 @@ API가 시스템 리소스 및 동작에 대한 명확하고 일관된 보기를
...
+따라서, 쿠버네티스는 쿠버네티스 API가 지속적으로 변경되고 성장할 수 있도록 설계했다.
+쿠버네티스 프로젝트는 기존 클라이언트와의 호환성을 깨지 _않고_ 다른 프로젝트가
+적응할 기회를 가질 수 있도록 장기간 해당 호환성을 유지하는 것을 목표로 한다.
+
+일반적으로, 새 API 리소스와 새 리소스 필드는 자주 추가될 수 있다.
+리소스 또는 필드를 제거하려면
+[API 지원 중단 정책](/docs/reference/using-api/deprecation-policy/)을 따라야 한다.
+
+쿠버네티스는 일반적으로 API 버전 `v1` 에서 안정 버전(GA)에 도달하면, 공식 쿠버네티스 API에
+대한 호환성을 유지하기 위해 강력하게 약속한다. 또한,
```
Comment (pjhwa) [inline content/ko/docs/concepts/overview/kubernetes-api.md:121]:
```suggestion
대한 호환성 유지를 강력하게 이행한다. 또한,
```

Comment (pjhwa) [reply content/ko/docs/concepts/overview/kubernetes-api.md]: https://github.com/kubernetes/website/pull/25598#discussion_r542160361 의 의견에 따라 동일하게 수정합니다.

**Comment by gochist** [comment]
@pjhwa 감사합니다! 

/lgtm

**Comment by ClaudiaJKang** [comment]
@pjhwa 감사합니다~ 

/approve

**Pull request merged**